cypress/support/fragments/data_import/mapping_profiles/newMappingProfile.js

```diff
@@ -281,30 +297,45 @@ export default {
     cy.wait(1000);
   },
 
-  addNote:(note) => {
+  addAdministrativeNote:(note) => {
     cy.do(Select({ name:'profile.mappingDetails.mappingFields[9].repeatableFieldAction' }).choose('Add these to existing'));
```

Comment:
wow, what a name of this select

Could you move it into const please? I mean this string: 'profile.mappingDetails.mappingFields[9].repeatableFieldAction'